extensions/sql-migration/src/wizard/loginSelectorPage.ts

```diff
@@ -102,14 +104,30 @@ export class LoginSelectorPage extends MigrationWizardPage {
 				return false;
 			}
 
-			if (this.migrationStateModel._loginMigrationModel.selectedWindowsLogins && !this.migrationStateModel._aadDomainName) {
+			if (this.migrationStateModel._loginMigrationModel.selectedWindowsLogins) {
+				if (!this.migrationStateModel._aadDomainName) {
```

Comment:
could we change these variable names as well? aadDomainName => entra id?

Reply:
For now I have made name change to Entra to the strings that are customer facing. Can take this up as a cleanup task later.